.pre-commit-config.yaml

```diff
@@ -1,12 +1,24 @@
 exclude: ^user_docs/(?!en/).*$
 
+# https://pre-commit.ci/
+# Configuration for Continuous Integration service
+ci:
+  # Can't run Windows scripts on Linux
+  # These also do not apply fixes, only run tests
+  skip: [scons-source, checkPot, unitTest, licenseCheck]
+  submodules: true
+  autoupdate_commit_msg: "Pre-commit autoupdate"
+  autofix_commit_msg: "Pre-commit autofix"
+
 repos:
 
 - repo: https://github.com/pre-commit/pre-commit-hooks
   rev: v4.6.0
   hooks:
     # Checks python syntax
     - id: check-ast
+      # File uses encoding not supported in Linux
```

Comment:
Ugh, I actually believe that comtypes should consider to fix this, enforcing UTF-8 for their generated modules. I filed https://github.com/enthought/comtypes/issues/592

Reply:
Thanks agreed

Reply:
Perhaps we should also consider adding encoding checks to pre-commit?